### ISSUE TITLE
[PRE REVIEW]: Discrete Laplace mixture model with applications in forensic genetics

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#694]
**Submitting author:** @mikldk (<a href="http://orcid.org/0000-0002-0234-0266">Mikkel Meyer Andersen</a>)
**Repository:** <a href="https://github.com/mikldk/disclapmix" target ="_blank">https://github.com/mikldk/disclapmix</a>
**Version:** v1.7
**Editor:** @pjotrp
**Reviewers:** @mikldk

**Author instructions**

Thanks for submitting your paper to JOSS @mikldk. The JOSS editor (shown at the top of this issue) will work with you on this issue to find a reviewer for your submission before creating the main review issue.

@mikldk if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by arfon** [comment]
:wave: @mgymrek - would you be willing to edit this submission for JOSS?

**Comment by mikldk** [comment]
@mgymrek / @arfon: Suggested reviewers: @wrathematics / @nhejazi .

**Comment by mikldk** [comment]
If @mgymrek is unavailable, maybe @biorelated could be editor instead, @arfon?

**Comment by leeper** [comment]
@tracykteal or maybe @karthik would be better editors than me for this one, I suspect.

**Comment by arfon** [comment]
@wrathematics & @nhejazi - would you be willing to review this submission for JOSS?

**Comment by nhejazi** [comment]
@arfon Happy to help if needed.

**Comment by arfon** [comment]
@nhejazi - great, thanks!

**Comment by arfon** [comment]
@pjotrp or @brainstorm - would you be willing to edit this submission for JOSS? @nhejazi has kindly offered to review this submission.

**Comment by brainstorm** [comment]
Yeah, I can do that, @arfon ;)

**Comment by arfon** [comment]
> I can also take this genetics contribution. @brainstorm if you are OK with that.

Perhaps one of you could review 😉

**Comment by brainstorm** [comment]
@pjotrp Would you like to review instead?

**Comment by pjotrp** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List all of Whedon's capabilities
@whedon commands

# Assign a GitHub user as the sole reviewer of this submission
@whedon assign @username as reviewer

# Add a GitHub user to the reviewers of this submission
@whedon add @username as reviewer

# Remove a GitHub user from the reviewers of this submission
@whedon remove @username as reviewer

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

# Change editorial assignment
@whedon assign @username as editor

# Set the software archive DOI at the top of the issue e.g.
@whedon set 10.0000/zenodo.00000 as archive

# Open the review issue
@whedon start review

🚧 🚧 🚧 Experimental Whedon features 🚧 🚧 🚧

# Compile the paper
@whedon generate pdf

```

**Comment by pjotrp** [comment]
@whedon assign @pjotrp as editor

**Comment by whedon** [comment]
OK, the editor is @pjotrp

**Comment by pjotrp** [comment]
Hi @mikldk, thanks for your interesting contribution. I'll look for a reviewer. Feel free to propose someone.

**Comment by mikldk** [comment]
Thanks, @pjotrp . And thank you for editing this. @nhejazi already agreed to do so (see above). I hope that he's still able to?

**Comment by pjotrp** [comment]
@whedon assign @mikldk as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @mikldk

**Comment by pjotrp** [comment]
@whedon list editors

**Comment by whedon** [comment]
Current editors:

```
@Kevin-Mattheus-Moerman
@acabunoc
@arfon
@arokem
@brainstorm
@cMadan
@danielskatz
@george-githinji
@jakevdp
@jasonclark
@karthik
@katyhuff
@kyleniemeyer
@labarba
@leeper
@lheagy
@mgymrek
@pjotrp
@tracykteal
```

**Comment by pjotrp** [comment]
@whedon start review magic-word=bananas

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/748. Feel free to close this issue now!

**Comment by pjotrp** [comment]
@whedon set @nhejazi as reviewer

**Comment by pjotrp** [comment]
@arfon I messed up. I set the author as reviewer and started review. The system does not like me to change anything now... Help!

**Comment by arfon** [comment]
@pjotrp - no problem, I've fixed it up in #748

**Comment by pjotrp** [comment]
Thanks @arfon, you are a pal!
